### PR TITLE
OLS-755: Add max_tokens_for_response as model parameters

### PR DESCRIPTION
**Pull request by onmete**
## Description

Add max_tokens_for_response as model parameters

## Type of change

- [x] New feature

### CODE REVIEW

**Comment by raptorsun** [comment]
The logics looks good. 
2 points:
1. remove the change on image reference, since it is not related to this functionality.
2. generate the bundle because the CRD is changed. use this command 
```
make opm
export PATH=$PATH:$PWD/bin
make update-bundle-catalog
```
the new changes in CRD should reflect in the ClusterServiceVersion resource in the bundle.

**Comment by raptorsun** [comment]
please add a unit test for the generated config file content with this new parameter, too.
The test is here https://github.com/openshift/lightspeed-operator/blob/main/internal/controller/ols_app_server_assets_test.go#L89

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -3,6 +3,6 @@ resources:
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
-- digest: sha256:e7c4661a256560de864a7076f66ec5729aa7d4617b93cb8804a5f4831ad31e6c
```
[the remark this answers is not in the document]
Comment (onmete) [reply config/manager/kustomization.yaml:6]: I've removed it, but it was autogenerated again after these commands
```sh
make opm
export PATH=$PATH:$PWD/bin
make update-bundle-catalog
```
Should I remove it again?

[the remark this answers is not in the document]
Comment (raptorsun) [reply config/manager/kustomization.yaml:6]: yes, the update catalog script needs some work to automatically compensate this.
Please revert this modification manually.

File: bundle/manifests/lightspeed-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -556,8 +562,8 @@ spec:
...
   relatedImages:
     - name: lightspeed-service-api
-      image: quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/lightspeed-service@sha256:de93445e0fa4c41df23e372ebed021277092f9dbbf6f88178447ccf69b9a6df6
+      image: quay.io/openshift-lightspeed/lightspeed-service-api:latest
     - name: lightspeed-console-plugin
-      image: quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/lightspeed-console@sha256:578ff950761cce1bf5baab046446317139efb65532f934623a171bf043be3775
+      image: quay.io/openshift-lightspeed/lightspeed-console-plugin:latest
     - name: lightspeed-operator
-      image: quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/lightspeed-operator@sha256:28884d235cc309f24a5f1e8ebb811ebfb900e55cdba8928a16c3f2f994a015d6
+      image: quay.io/openshift-lightspeed/lightspeed-operator:latest
```
Comment (raptorsun) [inline bundle/manifests/lightspeed-operator.clusterserviceversion.yaml:569]: please revert this piece manually, the script is not handling this yet.

File: bundle/manifests/lightspeed-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -362,7 +368,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
-                image: quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/lightspeed-operator@sha256:28884d235cc309f24a5f1e8ebb811ebfb900e55cdba8928a16c3f2f994a015d6
+                image: quay.io/openshift-lightspeed/lightspeed-operator:latest
```
Comment (raptorsun) [inline bundle/manifests/lightspeed-operator.clusterserviceversion.yaml:371]: revert this chunk manually, too :)

**Comment by xrajesh** [comment]
@onmete Could you please add the story that require this PR ?

**Pull request merged**